public/app/features/provisioning/RecentJobs.tsx

```diff
@@ -1,8 +1,10 @@
 import { useMemo } from 'react';
 
-import { intervalToAbbreviatedDurationString } from '@grafana/data';
+import { intervalToAbbreviatedDurationString, TraceKeyValuePair } from '@grafana/data';
 import { Spinner, Alert, Badge, InteractiveTable, Card, Box, Stack, Icon, Text, JSONFormatter } from '@grafana/ui';
 
+import KeyValuesTable from '../explore/TraceView/components/TraceTimelineViewer/SpanDetail/KeyValuesTable';
+
```

Comment:
^^ @Clarity-89 any better suggestions?  the trace view will format JSON elements 🤷🏻‍♂️

Reply:
Don't think so, or at least I'm not aware of any utils like this.